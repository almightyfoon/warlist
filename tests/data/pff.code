c8501b_-0p5C4QjGjGjFjFg4g4g45g5ghm5e5e5f5f___-0p5vjFjF4H4H4Hg45g5gfI0U5e5e5f5fi7
