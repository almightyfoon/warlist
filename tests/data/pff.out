https://conflictchamber.com/#c8501b_-0p5C4QjGjGjFjFg4g4g45g5ghm5e5e5f5f___-0p5vjFjF4H4H4Hg45g5gfI0U5e5e5f5fi7

Circle Army - 75 / 75 points
[Theme] The Bones of Orboros

[Baldur 2] Baldur the Stonesoul [+30]
 - Megalith [20]
 - Wold Guardian [16]
 - Wold Guardian [16]
 - Woldwarden [14]
 - Woldwarden [14]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [3]
Blackclad Stoneshaper [3]
Blackclad Wayfarer [0(4)]
Blackclad Wayfarer [0(4)]
Gremlin Swarm [3]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Shifting Stones [3]
Shifting Stones [3]

Circle Army - 75 / 75 points
[Theme] The Bones of Orboros

[Wurmwood 1] Wurmwood, Tree of Fate [+27]
 - Woldwarden [14]
 - Woldwarden [14]
 - Woldwyrd [9]
 - Woldwyrd [9]
 - Woldwyrd [9]
Blackclad Stoneshaper [3]
Blackclad Wayfarer [0(4)]
Blackclad Wayfarer [0(4)]
Bog Trog Mist Speaker [4]
Farrow Bone Grinders (min) [5]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Shifting Stones [3]
Shifting Stones [3]
Celestial Fulcrum [19]
